contracts/modules/v2/StakingPool.sol

```diff
@@ -103,6 +103,9 @@ contract StakingPool is ERC20 {
   uint public constant PARAM_PRECISION = 10_000;
   uint public constant BUCKET_SIZE = 7 days;
 
+  uint public constant PRICE_CURVE_EXPONENT = 7;
+  uint public constant MAX_PRICE_PERCENTAGE = 1e20;
```

Comment:
Rename pricePercentage to priceRatio and MAX_PRICE_PERCENTAGE to PRICE_DENOMINATOR